central/role/datastore/singleton.go

```diff
@@ -98,12 +98,12 @@ var defaultRoles = map[string]roleAttributes{
 		idSuffix:    "scopemanager",
 		description: "For users: use it to create and modify scopes for the purpose of access control or vulnerability reporting",
 		resourceWithAccess: []permissions.ResourceWithAccess{
-			permissions.View(resources.AuthProvider),
+			permissions.View(resources.Access),
 			permissions.View(resources.Cluster),
 			permissions.View(resources.Namespace),
-			permissions.View(resources.Role),
-			permissions.Modify(resources.Role),
-			permissions.View(resources.AuthProvider),
+			permissions.View(resources.Access),
+			permissions.Modify(resources.Access),
+			permissions.View(resources.Access),
```

Comment:
Please de-dupe, one `permissions.View(resources.Access),` is enough : )

Reply:
That is a good point. I had it on the radar.
There are a few cases where there were dupes before the change. Considering the order, I wonder if some resources were meant to be writeable. Now this is not the point of the change at hand. I'll do a deduplication pass (here, on the graphql resolver side, on some store cases where the grouped permission simplifies the access control, and on the existing unit test cases where test logic for the replacing resources was added).